buildcontext/parsefeatures.go

```diff
@@ -27,6 +28,15 @@ func parseFeatures(buildFilePath string, featureFlagOverrides string, projectRef
 		return nil, fmt.Errorf("No version specified in %s/Earthfile", projectRef)
 	}
 
+	warningStrs, err := ftrs.Adjust()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(warningStrs) > 0 {
+		console.Printf("Note that the feature flag %s is not necessary for version %s, since the feature was enabled by default. You may remove this flag from your Earthfile.", strings.Join(warningStrs, ", "), ftrs.Version())
```

Comment:
Let's reword this to be a bit shorter:
```suggestion
		console.Printf("NOTE: The %s feature is enabled by default under VERSION %s, and can be safely removed from the VERSION command", strings.Join(warningStrs, ", "), ftrs.Version())
```